CONTRIBUTORS.md

```diff
@@ -1,6 +1,6 @@
 # Contributors
 
-- Brian Bugert, Programming
+- Brian Bugert, Supreme Overlord
```

Comment:
I'd suggest "Supreme Overlord, Beneveolent Dictator of all life unholy"

Reply:
Hmmm, I think its limiting my power too much. I think not specifying 'unholy' leaves some room. However, I do like the term "Benevolent Dictator"....